region/maridia/inner-pink/Botwoon Hallway.json

```
@@ -405,6 +405,110 @@
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Suitless 10 Power Bomb Crystal Flash Clip (Left to Right)",
+      "requires": [
+        "canBeVeryPatient",
+        "canSuitlessMaridia",
+        {"or": [
+            "HiJump",
+            "canSpringBallJumpMidAir",
+            {
+              "and": [
+                "h_underwaterCrouchJumpDownGrab",
+                "canCarefulJump"
+            ]}
+        ]},
+        "h_10PowerBombCrystalFlash",
+        {"resetRoom": {"nodes": [1]}}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Lure a Puyo close to the crumble blocks and time it so you can kill it and collect its drop without getting hit.",
+        "Alternativly you can start with more health and time the PowerBomb explosion with a Puyo hit to collect the drop automatically."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "R-Mode Forced Standup Clip (Left to Right)",
+      "entranceCondition": {
+        "comeInWithRMode": {},
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        {"disableEquipment": "ETank"},
+        {"resourceAvailable": [{"type": "Energy", "count": 99}]},
+        "Gravity",
+        {"or": [
+            "canBeVeryPatient",
+            {"and": [
+                "Morph",
+                "canBePatient"
+            ]}
+        ]},
+        {"partialRefill": {"type": "ReserveEnergy"}},
+        {"autoReserveTrigger": {}}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Gravity suit allows easy traversal of the room and the ability to outrun the mochtroid to safely damage it down and get it into position",
+        "Perform a suitless crouch jump at the crumbles so that the reserve trigger forced standup happens just before Samus contacts the crumbles and still has upwards momentum.",
+        "Hold down during the forced standup until through the crumbles or they break. The Mochtroid must be killed quickly after the clip if the crumbles are broken.",
+        "If Morph is available a jump morph can be used for more leniency, giving a 9 frame window compared to 3 frames for the crouch jump."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Suitless R-Mode Forced Standup Clip (Left to Right)",
+      "entranceCondition": {
+        "comeInWithRMode": {},
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        {"disableEquipment": "ETank"},
+        {"resourceAvailable": [{"type": "Energy", "count": 99}]},
+        "canSuitlessMaridia",
+        "h_underwaterCrouchJumpDownGrab",
+        "canCarefulJump",
+        {"or": [
+            "canBeExtremelyPatient",
+            {"and": [
+                "Morph",
+                "canBeVeryPatient",
+            {"or": [
+                "Plasma",
+                "Wave",
+                {"ammo": {"type": "Super", "count": 1}},
+                {"ammo": {"type": "Missile", "count": 1}}
+            ]}
```

```suggestion
                {"or": [
                    "Plasma",
                    "Wave",
                    {"ammo": {"type": "Super", "count": 1}},
                    {"ammo": {"type": "Missile", "count": 1}}
                ]}
```
This looks placed incorrectly.  Since canBeVeryPatient is already very high tech.
And, do you think gravity or weapons to avoid the mochtroid are really necessary?